cluster/gce/container-linux/configure-helper.sh

```diff
@@ -360,6 +363,30 @@ current-context: service-account-context
 EOF
 }
 
+function create-kubescheduler-kubeconfig {
```

Comment:
working on a guide for deployers setting up control plane components to work out of the box with RBAC at https://docs.google.com/document/d/1PqI--ql3LQsA69fEvRq1nQWgiIoE5Dyftja5Um9ML7Q/edit

that will be pulled into the doc repo as soon as we have a way to open PRs for 1.6 doc content